OpenRA.Game/FileSystem/GlobalFileSystem.cs

```diff
@@ -185,10 +185,23 @@ public static Stream Open(string filename)
 			return s;
 		}
 
-		public static bool TryOpen(string filename, out Stream s)
+		public static bool TryOpen(string name, out Stream s)
 		{
-			// Check the cache for a quick lookup
-			if (filename.IndexOfAny(new char[] { '/', '\\' }) == -1)
+			var filename = string.Empty;
+			var foldername = string.Empty;
+
+			var explicitFolder = name.Contains(':');
+			if (explicitFolder)
+			{
+				var divide = name.Split(':');
+				foldername = divide.First();
+				filename = divide.Last();
+			}
+			else
+				filename = name;
+
+			// Check the cache for a quick lookup if the folder name is unknown
+			if (filename.IndexOfAny(new char[] { '/', '\\' }) == -1 && string.IsNullOrEmpty(foldername))
```

Comment:
This forces these faction-specific sprites to take the slow path, which kind of sucks.
The whole filesystem code needs a rewrite, so I won't block the pr over that, but please add a `// TODO` about this.